receiver/bigipreceiver/metadata.yaml

```diff
@@ -0,0 +1,278 @@
+  bigip.virtual_server.data.transmitted:
+    description: Amount of data transmitted to and from the virtual server.
+    unit: "By"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [direction]
+    enabled: true
+  bigip.virtual_server.connection.count:
+    description: Current number of connections to the virtual server.
+    unit: "{connections}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  bigip.virtual_server.request.count:
+    description: Number of requests to the virtual server.
+    unit: "{requests}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  bigip.virtual_server.packet.count:
+    description: Number of packets transmitted to and from the virtual server.
+    unit: "{packets}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [direction]
+    enabled: true
+  bigip.virtual_server.availability:
+    description: Availability of the virtual server.
+    unit: 1
+    gauge:
+      value_type: int
+    attributes: [availability.status]
+    enabled: true
+  bigip.virtual_server.enabled:
+    description: Enabled state of of the virtual server.
+    unit: 1
+    gauge:
+      value_type: int
+    attributes: [enabled.status]
+    enabled: true
+  bigip.pool.data.transmitted:
+    description: Amount of data transmitted to and from the pool.
+    unit: "By"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [direction]
+    enabled: true
+  bigip.pool.connection.count:
+    description: Current number of connections to the pool.
+    unit: "{connections}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  bigip.pool.request.count:
+    description: Number of requests to the pool.
+    unit: "{requests}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  bigip.pool.packet.count:
+    description: Number of packets transmitted to and from the pool.
+    unit: "{packets}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [direction]
+    enabled: true
+  bigip.pool.member.count:
+    description: Total number of pool members.
+    unit: "{members}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  bigip.pool.active_member.count:
+    description: Number of active pool members.
+    unit: "{members}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  bigip.pool.available_member.count:
+    description: Number of available pool members.
+    unit: "{members}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    enabled: true
```

Comment:
Does "active + available = total"?

Reply:
@djaglowski no, active is more about available + enabled i believe

Reply:
Is there a reasonable way to combine all this? I don't know exactly how they relate to each other, but it seems like `bigip.pool.member.count` could just have one or more attributes to differentiate the data points.

Reply:
@djaglowski so we had a discussion in slack, and it sounds like maybe these metrics don't belong lumped into one. they are definitely related, but they don't individually add together to make anything meaningful.
total = total amount of pools
available = available amount of pools (pools that appear to have valid configurations and "could" be used)
active = subset of available pools. the pools that are both available AND enabled.

so when you look at active pools and you look at available pools, you might be looking at the same pools (at least for some of them).  let me know if you still think strongly that we should lump these together even if the different groups dont' add up to a total

Reply:
@djaglowski changed these metrics to be a single one with an active attribute ("active"/"inactive"). removed available count as it doesn't seem to be as useful